internal/node/node.bzl

```diff
@@ -658,6 +661,27 @@ def nodejs_binary_macro(name, **kwargs):
         **kwargs
     )
 
+def nodejs_binary_toolchain_macro(name, toolchains = [], **kwargs):
```

Comment:
I wonder if this is holding it's weight and worth putting in rules_nodejs code, rather than just put it in an example and let users maintain a little macro for this purpose. In all the life of the project I think we only heard of two users who want it

Reply:
I'm not sure what the outcome here should be. Should it switch to adding nodejs_binary to rules_nodejs then have an example there? Or just remove this macro and create an example that that shows users how to do it?

My only push back on not many users requesting it is, has there been many build tools that can make it this easy to run different toolchains on the same machine in parallel. Could be useful during upgrades to different node versions and allow better gradual migrations.